src/main/java/com/auth0/json/mgmt/users/refreshtokens/RefreshToken.java

```diff
@@ -0,0 +1,97 @@
+package com.auth0.json.mgmt.users.refreshtokens;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Date;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class RefreshToken {
+    @JsonProperty("id")
+    private String id;
+    @JsonProperty("user_id")
+    private String userId;
+    @JsonProperty("created_at")
+    private Date createdAt;
+    @JsonProperty("idle_expires_at")
+    private Date idleExpiresAt;
+    @JsonProperty("expires_at")
+    private Date expiresAt;
+    @JsonProperty("client_id")
+    private String clientId;
+    @JsonProperty("session_id")
+    private String sessionId;
+    @JsonProperty("rotating")
+    private Boolean rotating;
+    @JsonProperty("resource_servers")
+    private List<ResourceServer> resourceServers;
```

Comment:
Device object and last_exchanged_at fields are missing. For reference - https://auth0.com/docs/api/management/v2/users/get-refresh-tokens-for-user#scopes